File: Sep_18/Count_Occurences.java
package CP_Club_SST.Sep_18;

public class Count_Occurences {
    public static void main(String[] args) {
        String s = "bobbobobobobobobboa";
        System.out.println(countOccurencesBob(s));
    }

    public static Integer countOccurencesBob(String s) {
        int count = 0;
        for (int i=0; i<(s.length()-2); i++) {
            if ((int)s.charAt(i) == 98 && (int)s.charAt(i+1) == 111 && (int)s.charAt(i+2) == 98) {
                i += 1;
                count++;
            }
        }
        return count;
    }
}

File: Sep_18/Count_Occurences.py
# Count the number of times "bob" occurs in a string A with all lowercase alphabets

def countOccurencesBob(A):
    count = 0
    i = 0
    for i in range(len(A)-2): # so we can look for i+1 and i+2
        if ord(A[i]) == 98 and ord(A[i+1]) == 111 and ord(A[i+2]) == 98:
            count += 1
            i += 1
    return count

print(countOccurencesBob("bobbobobobobobobboa"))